easybuild/easyconfigs/r/ROOT/ROOT-v6.10.08-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'CMakeMake'
+
+name = 'ROOT'
+version = 'v6.10.08'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://root.cern.ch/drupal/'
+description = """The ROOT system provides a set of OO frameworks with all the functionality
+    needed to handle and analyze large amounts of data in a very efficient way."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://root.cern.ch/download/']
+sources = ['%(namelower)s_%(version)s.source.tar.gz']
+checksums = ['2cd276d2ac365403c66f08edd1be62fe932a0334f76349b24d8c737c0d6dad8a']
+
+builddependencies = [
+    ('CMake', '3.9.5'),
+]
+
+dependencies = [
+    ('GSL', '2.4'),
+    ('libxml2', '2.9.6'),
+    ('PCRE', '8.41'),
+    ('CFITSIO', '3.42'),
+    ('freetype', '2.8.1'),
+    ('Python', '2.7.14'),
+    ('zlib', '1.2.11'),
+    ('X11', '20171023'),
+    ('Mesa', '17.2.5'),
+    ('libGLU', '9.0.0'),
+    ('GL2PS', '1.4.0'),
```

Comment:
missing `XZ`, `SQLite` deps? (`FFTW` is included in `foss` toolchain, so that's covered)?

cfr. #5370